constants.php

```diff
@@ -1,11 +1,16 @@
 <?php
-define('FRESHRSS_VERSION', '1.8.0');
+define('FRESHRSS_VERSION', '1.8.1-dev');
```

Comment:
Should be 1.9.0, right?

Reply:
Yes yes, I do that at the last minute before release :-)